onadata/apps/api/tests/viewsets/test_charts_viewset.py

```diff
@@ -47,6 +74,59 @@ def setUp(self):
                 os.path.dirname(__file__), '..', 'fixtures', 'forms',
                 'tutorial', 'instances', '3.xml'))
 
+    def test_correct_data_for_charts(self):
```

Comment:
Should we change the test name to something like `test_correct_merged_dataset_data_for_charts`?